IRIS-455 IRIS-456 checkResults() examples for wd and wdio

kb-kerem opened the pull request.

wswebcreation reviewed (commented):
Hi @kb-kerem 

This looks good, can you please let someone with more Java knowledge check the JAVA examples?

Regarding DWO, can you also update the docs and explain what the result of the check can be?

kb-kerem commented:
I've added the examples to readme files instead and removed the actual commands from the tests for the sake of keeping the tests simple @wswebcreation

wswebcreation reviewed (approved):
LGTM
